packages/vite/src/vite.ts

```diff
@@ -203,3 +203,17 @@ export const bundle: NuxtBuilder['bundle'] = async (nuxt) => {
   await buildClient(ctx)
   await buildServer(ctx)
 }
+
+export function mergeTransformAssetUrls (config: ViteConfig) {
+  config.vue = vite.mergeConfig(config.vue || {}, {
+    template: {
+      transformAssetUrls: {
+        video: ['src', 'poster'],
+        source: ['src'],
+        img: ['src'],
+        image: ['xlink:href', 'href'],
+        use: ['xlink:href', 'href']
+      }
+    }
```

Comment:
how about we add these here instead?

https://github.com/nuxt/nuxt/blob/4f017a5538d1c17864fca5723c5021f13cd9723f/packages/vite/src/vite.ts#L113

Reply:
sure, it makes more sense